Modules/fcntlmodule.c

```diff
@@ -324,6 +324,11 @@ fcntl_flock_impl(PyObject *module, int fd, int code)
     }
 #endif /* HAVE_FLOCK */
     if (ret < 0) {
+#if defined(_AIX)
+        if (code & LOCK_NB && errno == EACCES) {
+            errno = EWOULDBLOCK;
```

Comment:
This change should be applied to lockf as well I think.

Reply:
Fixed in eb62971